functions/__fzf_grep_current_dir.fish

```diff
@@ -0,0 +1,40 @@
+function __fzf_grep_current_dir --description "Search current directory content for $token. Replace the current token with the selected file paths."
+    # Make sure that fzf uses fish to execute __fzf_preview_file.
+    # See similar comment in __fzf_search_shell_variables.fish.
+    set --local --export SHELL (command --search fish)
+
+    set rg_cmd rg --ignore-case --files-with-matches
+    set redirect "< /dev/tty > /dev/tty 2>&1"
+
+    set token (commandline --current-token | string unescape)
+
+    set fzf_arguments --multi --ansi --exact --phony --query=$token \
```

Comment:
Better remove `--ansi` because there's no color (and `rg --color=always` gives a single color, which isn't useful too).